src/nvim/message.c

```diff
@@ -572,8 +564,22 @@ int emsg(const char_u *s_)
 
   // Display the error message itself.
   msg_nowait = false;  // Wait for this msg.
-  return msg_attr(s, attr);
+  return (*msg_attr)(s, attr);
+}
+
+/*
+ * emsg() - display an error message
+ *
+ * Rings the bell, if appropriate, and calls message() to do the real work
+ * When terminal not initialized (yet) mch_errmsg(..) is used.
+ *
+ * return TRUE if wait_return not called
+ */
+int emsg(const char_u *s_)
```

Comment:
No need to have `const char *s = (const char *)s_;` line above as well as underscore here: underscore is only because of the existence of the line and line is there only because I did not feel like refactoring lots of `emsg` callers which use `char_u`.

Reply:
Ah, and here still is underscore.

Reply:
Oh god, I must be blind. Sorry for that, I will change it immediately so I will not forget anymore !